packages/mattermost-redux/src/actions/search.ts

```diff
@@ -16,7 +16,7 @@ import {FileSearchResults, FileSearchResultItem} from '@mattermost/types/files';
 
 import {SearchParameter} from '@mattermost/types/search';
 
-import {getChannelAndMyMember, getChannelMembers} from './channels';
+import {getChannelAndMyMember, getChannelMembers, getChannelStats} from './channels';
```

Comment:
@koox00 or @AshishDhama was this the API call that breaks the server?

Reply:
Yup, this is the one that breaks the server.

Reply:
OK, in this case we should wait for the fix on that to be merged and continue afterwards. @AshishDhama could you link/mention the other PR here so we can keep track? Thanks!

Reply:
@michelengelen @ashishbhate will work on the backend but yeah I will post link as soon as we have it

Reply:
What's the ticket for that one?

Reply:
https://mattermost.atlassian.net/browse/MM-49486

Reply:
PR is up for that https://github.com/mattermost/mattermost-server/pull/22096

Reply:
@AshishDhama Resolving this since we're no longer making that extra request at all